### ISSUE TITLE
Nuxt TypeScript error: nuxt:typescript Cannot find module '@/my-module'

### ISSUE THREAD

**Issue by dugajean** [nuxt/typescript#157]
**Describe the bug**
I have setup Nuxt with TypeScript using the instructions from https://typescript.nuxt.org. The transition was pretty seamless and ok, except that I can't get rid of this error that `nuxt:typescript` "Cannot find module", which obviously is a false-positive.

My imports look like this:

```
import InputField from '@/components/InputField.vue'
```

I have tried with `~/` and without the `.vue` extension. Nothing works.

My `tsconfig.json` looks like this:

```
{
  "compilerOptions": {
    "target": "esnext",
    "module": "esnext",
    "moduleResolution": "node",
    "lib": ["esnext", "esnext.asynciterable", "dom"],
    "esModuleInterop": true,
    "allowJs": true,
    "sourceMap": true,
    "strict": true,
    "noEmit": true,
    "baseUrl": ".",
    "paths": {
      "~/*": ["src/*"],
      "~*": ["src/*"],
      "@/*": ["src/*"]
    },
    "types": ["@types/node", "@nuxt/types"]
  },
  "exclude": ["node_modules"]
}
```

And I have this object in my `nuxt.config.js`:

```
typescript: {
  typeCheck: {
    eslint: true,
    vue: true
  }
}
```
**To Reproduce**
1. Install Nuxt and its Nuxt TS (incl. runtime)
2. Import your modules via `~/` or `@/`
3. CLI error shows up, but the site works

**Expected behavior**
There should not be an error, even though this error isn't blocking anything.

**Screenshots**
![image](https://user-images.githubusercontent.com/2448254/67163205-7830ea80-f36c-11e9-9a33-464539110239.png)

**Comment by kevinmarrec** [comment]
Duplicate of https://github.com/nuxt/typescript/issues/153

**Comment by dugajean** [comment]
@kevinmarrec There's another problem with `.ts` files. Is that a new issue?

https://github.com/nuxt/typescript/issues/153#issuecomment-544279653

**Comment by kevinmarrec** [comment]
@dugajean No, the author removed his comment, I never had problem with `.ts` files ^^

**Comment by jggj21** [comment]
How did you solve it?
